### ISSUE TITLE
When will the image of NanoPi M6 be released?

### ISSUE THREAD

**Issue by LeeXujie** [Joshua-Riek/ubuntu-rockchip#1044]

**Comment by Joshua-Riek** [comment]
This is one of the more interesting boards because of the built in display, I need to get my hands on the hardware then work on drivers.

**Comment by LeeXujie** [comment]
> This is one of the more interesting boards because of the built in display, I need to get my hands on the hardware then work on drivers.

Yes, their development board hardware is indeed doing well, but their official system seems to have poor GPU support. When I run the VTK program on Ubuntu 22.04, the interactive interface is very laggy. Can you give some suggestions to solve this problem?

![screenshot2024-0918](https://github.com/user-attachments/assets/75dcdb9d-b1ac-4fd2-a0cd-f324f02c7fd7)

**Comment by lukaszsobala** [comment]
@LeeXujie Just add a ppa with panfork mesa drivers, for example this one, and install drivers that actually work for rk3588.

https://launchpad.net/~jjriek/+archive/ubuntu/panfork-mesa/
